src/coreclr/jit/lsrabuild.cpp

```diff
@@ -1899,6 +1899,19 @@ void LinearScan::buildPhysRegRecords()
         RegRecord* curr = &physRegs[reg];
         curr->regOrder  = (unsigned char)i;
     }
+#if defined(TARGET_AMD64)
+    if (compiler->canUseEvexEncoding())
+    {
+        const regNumber lsraRegOrderFltUpper[]   = {REG_VAR_ORDER_FLT_UPPER};
```

Comment:
Should this be `static const` to match the definition of `lsraRegOrderFlt`?

Should it be defined on L1878 as a general global alongside the other two? -- AFAIR the space is allocated regardless and the only benefit of defining it in a function is that it won't be initialized until this block is hit the first time. However, given it is all constant data, I don't think there is any real benefit to doing that.

Reply:
comment resolved.